lib/fluent/command/ctl.rb

```diff
@@ -44,13 +45,15 @@ class Ctl
         restart: 128,
         flush: 129,
         reload: SERVICE_CONTROL_PARAMCHANGE,
+        # dump: 130? TODO support svc control
       }
     else
       COMMAND_MAP = {
         shutdown: :TERM,
         restart: :HUP,
         flush: :USR1,
         reload: :USR2,
+        dump: "CONT",
```

Comment:
Should we use `symbol` style here?

Reply:
Thanks! I have unified by `symbol` here.